src/Components/App/App.js

```diff
@@ -55,9 +49,7 @@ const App = () => {
 // }
 
   const removeFav =  (truck) => {
-
-  const newFavState = favorites.filter((fav) => fav.id !== truck.id)
-  setFavorites(newFavState)
+    setVendors(vendors.filter((v) => v.favorited === true))
```

Comment:
This is nice for when we are in favorites view. Will be interesting to make it not effect the other vendors when in the main page.